### DISCUSSION TITLE
Is the GeForce GT 635M not compatible with upscayl?

### DISCUSSION THREAD

**Discussion by snel0026** [upscayl/upscayl#810] [Q&A]
Is the GeForce GT 635M not compatible with Upscayl?
I got the following log:

⚙️ Setting saveImageAs to png
🔀 Setting model to realesrgan-x4plus
⚙️ Setting gpuId to empty string
🔀 Setting model to 
🔀 Setting model to realesrgan-x4plus
🔄 Resetting image paths
🎒 BACKEND REPORTED:  🖼️ Updating Image Path:  C:\Users\Wim\Desktop\Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b.jpg
🎒 BACKEND REPORTED:  📄 Selected File Path:  C:\Users\Wim\Desktop\Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b.jpg
🖼 Selected Image Path:  C:\Users\Wim\Desktop\Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b.jpg
📁 Selected Image Directory:  C:\Users\Wim\Desktop
🖼 imagePath:  C:\Users\Wim\Desktop\Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b.jpg
🔤 Extension:  jpg
🔄 Resetting Upscaled Image Path
🏁 UPSCAYL
🎒 BACKEND REPORTED:  ✅ Upscayl Variables:  {"model":"realesrgan-x4plus","gpuId":null,"saveImageAs":"png","inputDir":"C:\\Users\\Wim\\Desktop","fileNameWithExt":"Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b.jpg","outputDir":"C:\\Users\\Wim\\Desktop","outFile":"C:\\Users\\Wim\\Desktop\\Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b_upscayl_4x_realesrgan-x4plus.png","fileName":"Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b","scale":"4","compression":"0","customWidth":"","useCustomWidth":false,"tileSize":null}
🎒 BACKEND REPORTED:  📢 Upscayl Command:  -i,C:\Users\Wim\Desktop\Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b.jpg,-o,C:\Users\Wim\Desktop\Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b_upscayl_4x_realesrgan-x4plus.png,-m,C:\Program Files\Upscayl\resources\models,-n,realesrgan-x4plus,-f,png,-c,0
🎒 BACKEND REPORTED:  👶 Updating Child Processes:  {"binary":"C:\\Program Files\\Upscayl\\resources\\bin\\upscayl-bin","args":["C:\\Program Files\\Upscayl\\resources\\bin\\upscayl-bin","-i","C:\\Users\\Wim\\Desktop\\Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b.jpg","-o","C:\\Users\\Wim\\Desktop\\Afbeelding van WhatsApp op 2024-05-03 om 16.14.03_99abba9b_upscayl_4x_realesrgan-x4plus.png","-m","C:\\Program Files\\Upscayl\\resources\\models","-n","realesrgan-x4plus","-f","png","-c","0"]}
🎒 BACKEND REPORTED:  🛑 Updating Stopped:  false
🎒 BACKEND REPORTED:  vkEnumeratePhysicalDevices failed -3
🚨 Error: Invalid GPU Device

🔄 Resetting image paths
🚧 UPSCAYL_PROGRESS:  vkEnumeratePhysicalDevices failed -3
🚨 Error: Invalid GPU Device

**Comment by aaronliu0130** [comment]
It was released before Vulkan, a technology we rely on, existed and https://www.techpowerup.com/gpu-specs/geforce-gt-635m.c352 says it has no Vulkan, so no, probably not.

**Comment by snel0026** [comment]
Hi Aaron,

Thanks, so i have to look for a new laptop.

Wim

 

Van: Aaron Liu ***@***.***> 
Verzonden: vrijdag 3 mei 2024 22:21
Aan: upscayl/upscayl ***@***.***>
CC: snel0026 ***@***.***>; Author ***@***.***>
Onderwerp: Re: [upscayl/upscayl] Is the GeForce GT 635M not compatible with upscayl? (Discussion #810)

 

It was released before Vulkan, a technology we rely on, existed and https://www.techpowerup.com/gpu-specs/geforce-gt-635m.c352 says it has no Vulkan, so no, probably not.
